gno.land/pkg/gnoweb/handler.go

```diff
@@ -183,6 +187,103 @@ func (h *WebHandler) GetRealmView(gnourl *weburl.GnoURL) (int, *components.View)
 	})
 }
 
+// GetUserView returns the user profile view for a given GnoURL.
+func (h *WebHandler) GetUserView(gnourl *weburl.GnoURL) (int, *components.View) {
+	username := strings.TrimPrefix(gnourl.Path, "/u/")
+	handlename := username
+
+	contributions := []components.UserContribution{
+		{
+			Title:       "gno-blogpost",
+			Description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
```

Comment:
I think most packages won't have a description. We may need some of them to specify a description, possibly with comments. However, consider that this will be very irregular. I suggest you plan for the following now:  
-  No description  
-  Single-line description  
-  Multi-line description to be stripped